### PR TITLE
Add ability to define annotations for web/worker Service accounts

### LINKED ISSUE #173: Allow for annotations to be assigned to service accounts
I have a use case whereby I need to assign annotations to the service account users that Concourse creates, currently the only workaround is to create your own service accounts and update the values so Concourse will know to use the custom ones instead.  It would be a lot nicer if this was supported as a feature out of the box.

### PR DESCRIPTION
**Pull request by Caprowni**
# Existing Issue

Fixes #173 

# Changes proposed in this pull request

* Adds annotations variables for both worker and web since they might have different use cases for annotations, these are defined inside the values.yaml and will be rendered(if there are any) into chart at runtime.
* Adds the annotation include statements into the web and worker service account template files. 

# Contributor Checklist

- [x] Variables are documented in the README.md
- [x] Which branch are you merging into?
    - `master` is for changes related to the current release of the `concourse/concourse:latest` image and should be good to publish immediately

Merging to master should be okay (?) happy to move to dev if need be, as I mentioned i've already worked around this by creating my own service accounts but would be nice to have this natively supported.

# Reviewer Checklist
> This section is intended for the core maintainers only, to track review progress. Please do not
> fill out this section.
- [ ] Code reviewed
- [ ] Topgun tests run
- [ ] Back-port if needed
- [ ] Is the correct branch targeted? (`master` or `dev`)

### CODE REVIEW

File: templates/web-serviceaccount.yaml [hunk 1]
```diff
@@ -9,5 +9,9 @@ metadata:
     chart: {{ .Chart.Name }}-{{ .Chart.Version }}
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
+    {{- if .Values.rbac.webServiceAccountAnnotations }}
+    annotations:
+{{ toYaml .Values.rbac.webServiceAccountAnnotations | indent 4 }}
+  {{- end -}}
```
Comment (YoussB) [inline templates/web-serviceaccount.yaml:15]:
```suggestion
    {{- end -}}
```
just a minor indentation

File: templates/worker-serviceaccount.yaml [hunk 2]
```diff
@@ -9,5 +9,9 @@ metadata:
     chart: {{ .Chart.Name }}-{{ .Chart.Version }}
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
+{{- if .Values.rbac.workerServiceAccountAnnotations }}
```
Comment (YoussB) [inline templates/worker-serviceaccount.yaml:12]:
```suggestion
    {{- if .Values.rbac.workerServiceAccountAnnotations }}
```

**Review by YoussB — CHANGES_REQUESTED** [review]
Thanks @Caprowni.
The PR looks good to me, I added a couple of small indentation suggestions.
Also, can you add the new parameter to `values.yml`

**Comment by Caprowni** [comment]
@YoussB Hey! Thanks for reviewing my PR, I fixed the indentation you mentioned, i'm not sure about the values.yaml comment though as i've already added the definitions into the values.yaml at the root of the repo.

Is there something i've missed?

**Review by YoussB — APPROVED** [review]
@Caprowni, sorry I probably missed that. I think this is good to go.

**Pull request merged**